driver/driver_default.go

```diff
@@ -14,12 +14,22 @@ type DefaultDriver struct {
 	r Registry
 }
 
-func NewDefaultDriver(l logrus.FieldLogger, version, build, date string) (Driver, error) {
+func NewDefaultDriver(l logrus.FieldLogger, version, build, date string, dev bool) (Driver, error) {
 	if l == nil {
 		l = logrusx.New()
 	}
 
-	c := configuration.NewViperProvider(l)
+	if dev {
```

Comment:
Let's move this warning to cmd/daemon instead - otherwise we'll get a lot of warnings during testing.

Reply:
The logger is not initialized in the deamon yet but that should be possible

Reply:
You can call it in "ServeAll", then it will be initialized